### PR TITLE
Define cluster_image here instead of cluster_infra.

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

File: group_vars/openstack [hunk 1]
```diff
@@ -16,6 +16,7 @@ openstack_auth:
   password: "{{ lookup('env', 'OS_PASSWORD') }}"
   auth_url: "{{ lookup('env', 'OS_AUTH_URL') }}"
 
+cluster_image: d7df64a3-9b2e-4ff2-820f-4481671d1dcf
```
Comment (oneswig) [inline group_vars/openstack]: Use the name, `centos-7-20190104`- do we really need the UUID instead?

**Review by oneswig — COMMENTED** [review]
Do we need to refer to the image by UUID?  I think the name is more friendly...

File: group_vars/openstack (hunk 1 above)
Comment (brtkwr) [reply group_vars/openstack]: Ack.

**Review by oneswig — APPROVED** [review]
Awesome

**Pull request merged**